src/pages/details/awsDetails/detailsTable.styles.ts

```diff
@@ -1,73 +0,0 @@
-import global_BackgroundColor_light_100 from '@patternfly/react-tokens/dist/js/global_BackgroundColor_light_100';
-import global_danger_color_100 from '@patternfly/react-tokens/dist/js/global_danger_color_100';
-import global_disabled_color_100 from '@patternfly/react-tokens/dist/js/global_disabled_color_100';
-import global_FontSize_xs from '@patternfly/react-tokens/dist/js/global_FontSize_xs';
-import global_spacer_3xl from '@patternfly/react-tokens/dist/js/global_spacer_3xl';
-import global_spacer_xs from '@patternfly/react-tokens/dist/js/global_spacer_xs';
-import global_success_color_100 from '@patternfly/react-tokens/dist/js/global_success_color_100';
-import { css } from 'emotion';
-import React from 'react';
-
-export const styles = {
-  emptyState: {
-    backgroundColor: global_BackgroundColor_light_100.value,
-    display: 'flex',
-    justifyContent: 'center',
-    paddingTop: global_spacer_3xl.value,
-    height: '35vh',
-    width: '100%',
-  },
-  infoArrow: {
-    position: 'relative',
-  },
-  infoArrowDesc: {
-    bottom: global_spacer_xs.value,
-  },
-  infoDescription: {
-    color: global_disabled_color_100.value,
-    fontSize: global_FontSize_xs.value,
-  },
-} as { [className: string]: React.CSSProperties };
```

Comment:
We don't need to convert these properties, since it does not use Emotion. Only the styles wrapped with `css` need to be converted. It can be implemented either way, but just more to test visually.

The Azure and OpenShift tables will also have a similar "styles" implementation. For consistency, it may be best to revert this one?

Reply:
I can revert these changes, I know when we talked that we  were just interested in changing over the emotion stuff. In this case I thought it was going to be cleaner not to have both files.

Reply:
Considering the we had a couple CSS bugs that needed to be fixed, I would prefer to limit the changes to just Emotion for now. This would help eliminate any surprises in production.